whatshap/variants.py

```diff
@@ -302,7 +308,7 @@ def _usable_alignments(self, chromosome, sample, regions=None):
     def has_reference(self, chromosome):
         return self._reader.has_reference(chromosome)
 
-    def _alignments_to_reads(self, alignments, variants, sample, reference):
+    def _alignments_to_reads(self, alignments, variants, sample, reference, genotypes):
```

Comment:
See comment above. Would be good to have an explanation why a genotype is passed here.